app/build.gradle

```diff
@@ -36,6 +36,9 @@ repositories {
     maven {
         url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/'
     }
+	maven {
+        url "https://dl.bintray.com/sufficientlysecure/maven"
```

Comment:
I just requested the inclusion into JCenter. Then this is no longer necessary.

Reply:
Thanks, I will monitor the availability and remove the repo inclusion when available.

Reply:
It's now on JCenter, my request has been approved.

Reply:
Thanks, I have removed the superfluous maven repo.